lms/config/__init__.py

```diff
@@ -36,8 +36,12 @@ def configure(settings):
         "h_jwt_client_secret": env_setting("H_JWT_CLIENT_SECRET", required=True),
         # The authority that we'll create h users and groups in (e.g. "lms.hypothes.is").
         "h_authority": env_setting("H_AUTHORITY", required=True),
-        # The base URL of the h API (e.g. "https://hypothes.is/api).
-        "h_api_url": env_setting("H_API_URL", required=True),
+        # The public base URL of the h API (e.g. "https://hypothes.is/api).
+        "h_api_url_public": env_setting("H_API_URL_PUBLIC", required=True),
+        # A private (within-VPC) URL for the same h API. Faster and more secure
+        # than the public one. This is used for internal server-to-server
+        # comms.
+        "h_api_url_private": env_setting("H_API_URL_PRIVATE", required=True),
```

Comment:
Right now, both of the envs (PUBLIC and PRIVATE) are required. Would it make sense to make one of them default to the value of the other? Or is making the app crash if either is missing a desired effect?

Reply:
Just requiring both is intended. The only two envs that app gets deployed to have both (our own qa and prod envs) and dev envs shouldn't differ from qa/prod. So fallback logic would never be used